tools/assignmentPlacer.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
+import shutil
+
+gradeable_path = '/var/local/submitty/courses/s20/sample/submissions/grades_released_homework_autohiddenEC'
+
+if __name__ == "__main__":
+    if not os.path.isdir('assignments') or not os.path.isdir(gradeable_path):
```

Comment:
Let's use an absolute path to this folder instead of it being relative (assuming we expect to always pick up out of `${lichen_installation_dir}/tools/assignments`.

Can do something like:
```
import pathlib
assignment_path = str(pathlib.Path(__file__).parent.absolute() / 'assignments')
```